tools/devops/automation/run-nightly-codeql.yml

```diff
@@ -0,0 +1,87 @@
+# xamarin-macios-nightly-codeql
+
+trigger: none
+pr: none
+
+schedules:
+- cron: "0 5 * * *"
+  displayName: Run daily at 5:00 UTC
+  branches:
+    include:
+    - main
```

Comment:
Should we only include main or the release branches too?

Reply:
I would assume main only - otherwise we'll have to backport any CodeQL fixes to all current release branches (otherwise we'll get AzDo issued filed/reopened).

Reply:
Our release branches are cut from main so as long as we keep main compliant we should be golden